From b6e48541c1aa4b07a0f5610bd4ed26d2d7ad78f6 Mon Sep 17 00:00:00 2001
From: Jordan Lazare <Jordan.Lazare@amd.com>
Date: Fri, 1 Apr 2016 12:44:04 -0400
Subject: [PATCH 0999/1110] drm/amd/dal: Revert disabling stream when clock
 source changes

It causes other issues when timing doesn't change where the pipe is never re-enabled

Signed-off-by: Jordan Lazare <Jordan.Lazare@amd.com>
Acked-by: Harry Wentland <harry.wentland@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/dal/dc/dce110/dce110_hw_sequencer.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/dal/dc/dce110/dce110_hw_sequencer.c b/drivers/gpu/drm/amd/dal/dc/dce110/dce110_hw_sequencer.c
index 47f75ef..c2adf6b 100644
--- a/drivers/gpu/drm/amd/dal/dc/dce110/dce110_hw_sequencer.c
+++ b/drivers/gpu/drm/amd/dal/dc/dce110/dce110_hw_sequencer.c
@@ -1197,8 +1197,6 @@ static enum dc_status apply_ctx_to_hw(
 		if (pipe_ctx_old->stream && !pipe_ctx->stream)
 			reset_single_pipe_hw_ctx(
 				dc, pipe_ctx_old, &dc->current_context);
-		else if (pipe_ctx_old->clock_source != pipe_ctx->clock_source)
-			core_link_disable_stream(pipe_ctx);
 	}
 
 	/* Skip applying if no targets */
-- 
2.7.4

